pkg/tfgen/docs.go

```diff
@@ -440,7 +442,7 @@ var (
 	)
 
 	attributeBulletRegexp = regexp.MustCompile(
-		"^\\s*[*+-]\\s*`([a-zA-z0-9_]*)`\\s*[:–-]?\\s*(.*)",
+		"^\\s*[*+-]\\s*`([a-zA-z0-9._]*)`\\s*[:–-]?\\s*(.*)",
```

Comment:
I think we are at the point where we need to break up this regex into multiple constants so we can name them:

```suggestion
		bulletPointRegexp + "`" + pathNameRegexp + `"` + "\\s*[:–-]?\\s*(.*)", // I'm not sure what the last section is.
```